ai_presenter/database.py

```diff
@@ -12,17 +13,34 @@ def __init__(self, data):
         self.voice_type = data['voice_type']
         self.age = data['age']
         self.height = data['height']
-        
+        self.gender = data['gender']
+
+
 class Scene:
     def __init__(self, data):
         self.name = data['name']
         self.location = data['location']
         self.actors = data['actors']
         self.dialogue = data['dialogue']
-        
+
 
 class Location:
     def __init__(self, data):
         self.name = data['name']
         self.type = data['type']
-        self.lighting = data['lighting']
\ No newline at end of file
+        self.lighting = data['lighting']
+
+
+class Config:
```

Comment:
Move Config classes to `config/` directory and have `config.py`, `ai.py`, and `chatgpt.py`